### PR TITLE
Update Flag

### PR DESCRIPTION
**Pull request by Lutzifer**
For me, it only worked with "=" instead of ":" here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ $ make
 ```sh
 $ git clone https://github.com/yonaskolb/Mint.git
 $ cd Mint
-$ swift run mint install yonaskolb/mint --global
+$ swift run mint install yonaskolb/mint --
```
Comment (yonaskolb) [inline README.md]: Was this an intended change?

Comment (yonaskolb) [reply README.md]: I guess this was trying to remove the global flag, but missed the —
I’ve updated this in master

**Comment by yonaskolb** [comment]
Thanks for the changes @Lutzifer! I’ve made them  in master as I didn’t hear back

File: README.md (hunk 1 above)
Comment (Lutzifer) [reply README.md]: ah, sorry, did not see the second part in the GitHub mail. This was indeed not intended, sry.

**Pull request closed without merging**